docs/modules/transform-sync/overview.md

```diff
@@ -1,3 +1,5 @@
+<%(TOC)%>
```

Comment:
I don't think we need a ToC on this page

Reply:
trying to be consistent with a couple of other modules - but probably makes sense to have no ToC on all of the module overview pages